superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -481,6 +497,27 @@ export default class ResultSet extends React.PureComponent<
     return <div className="noControls" />;
   }
 
+  rowsReturned() {
```

Comment:
nit, but I would rename this `renderRowsReturned` since it's a function, so thus the verb prefix.